Makefile

```diff
@@ -60,7 +64,7 @@ push_translations:
 	tx push -s
 
 # extract, compile, and check if translation files are up-to-date
-validate_translations: extract_translations compile_translations generate_dummy_translations detect_changed_source_translations
+check_translations_up_to_date: extract_translations compile_translations generate_dummy_translations detect_changed_source_translations test_translations
```

Comment:
Should `test_translations` still be in here?

Reply:
no. sorry my bad.